fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1279,6 +1279,14 @@ Future< Standalone<RangeResultRef> > ReadYourWritesTransaction::getRange(
 		}
 	}
 	
+	if (begin.getKey() == LiteralStringRef("\xff\xff/conflicting_keys") && end.getKey() == LiteralStringRef("\xff\xff/conflicting_keys\xff")) {
```

Comment:
This is a bit weird: this will only allow me to fetch the whole list in one query. But I think the following should also work:

```
tr.getRange(LiteralStringRef("\xff\xff/conflicting_keys/bar"), LiteralStringRef("\xff\xff/conflicting_keys/bar"));
```

Reply:
I have changed it to the way we discussed and add comments in the code.